dags/aggregates_python.py

```diff
@@ -29,24 +27,7 @@ def main_of_unzipped_data_uploader(ti):
     # Path to the CSV file containing data to be validated
     csv_file_path = 'unpacked_zip_data_files/wynik_zgloszenia_2022_up.csv'
 
-    # Database connection parameters
```

Comment:
is the "ti" argument is needed in this fucntion?

Reply:
I will check if it is necessary - if not, I will remove, I the opposite - I will leave it.